spec/android_update_release_notes_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::AndroidUpdateReleaseNotesAction do
+  let(:release_notes_txt) { File.join(File.dirname(__FILE__), 'RELEASE-NOTES.txt') }
+  let(:changelog_md) { File.join(File.dirname(__FILE__), 'CHANGELOG.md') }
+
+  after do
+    FileUtils.rm([release_notes_txt, changelog_md], force: true)
+  end
```

Comment:
Please don't make the tests write in the directory of the repo itself.

It's better to use `Dir.mktmpdir` to create a temporary directory (somewhere in `/tmp`, the method will take care of using the system temp dir and find a unique name for that) and write the temp file in it.

We also have `in_tmp_dir` and `with_tmp_file` helpers defined in `spec_helper.rb` that will surely help there.

Reply:
Done in 6baf51dbd13b8667d9503cb60c63ad399b19f47b and 9d4b18a2ff1b408150c79588a89dd315392a6956